apps/prairielearn/src/server.js

```diff
@@ -1543,8 +1543,6 @@ export async function initExpress() {
     (await import('./middlewares/logPageView.js')).default('studentAssessments'),
     (await import('./pages/studentAssessments/studentAssessments.js')).default,
   ]);
-  // Exam/Homeworks student routes are polymorphic - they have multiple handlers, each of
-  // which checks the assessment type and calls next() if it's not the right type
```

Comment:
Not directly related, but since I noticed this while searching, I removed it. The "polymorphic" handlers are no longer being used.